Tools/DeployApps/BaasClient.cs

```diff
@@ -76,25 +76,22 @@ public enum Result
                 };";
 
         private const string TriggerClientResetOnSyncServerFuncSource =
-            @"exports = async function(useId, appId = '') {
+            @"exports = async function(userId, appId = '') {
                 const mongodb = context.services.get('BackingDB');
                 console.log('user.id: ' + context.user.id);
-                let deletionResult;
                 try {
                   let dbName = '__realm_sync';
                   if (appId !== '')
                   {
                     dbName = [dbName, '_', appId].join('');
                   }
-                  deletionResult = await mongodb.db(dbName).collection('clientfiles').deleteMany({ ownerId: useId });
+                  const deletionResult = await mongodb.db(dbName).collection('clientfiles').deleteMany({ ownerId: userId });
                   console.log('Deleted documents: ' + deletionResult.deletedCount);
+
+                  return { status: deletionResult.deletedCount > 0 ? 'success' : 'failure' };
                 } catch(err) {
                   throw 'Deletion failed: ' + err;
                 }
-                if (deletionResult.deletedCount > 0) {
```

Comment:
How do we know if anything was deleted? If we don't care, why?

Reply:
I just moved the return inside the `try` block:

```
return { status: deletionResult.deletedCount > 0 ? 'success' : 'failure' };
```

Reply:
eh... completely blind... Sorry